frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx

```diff
@@ -65,7 +76,7 @@ export const FeatureStrategyMenu = ({
                 permission={CREATE_FEATURE_STRATEGY}
                 projectId={projectId}
                 environmentId={environmentId}
-                onClick={() => navigate(createStrategyPath)}
+                onClick={onClick}
```

Comment:
what about giving it a descriptive name so that I know when business operation onClick performs?

Reply:
I see similar naming anti-pattern in other places in this file. e.g. onClick={onClose}. I'd expect onClick={businessOperationName}